.nuget/packages.config

```diff
@@ -2,9 +2,10 @@
 <packages>
   <package id="ILMerge" version="2.14.1208" targetFramework="net45" />
   <package id="Microsoft.Build" version="15.1.262-preview5" targetFramework="net45" />
-  <package id="Microsoft.VisualStudio.Composition" version="15.0.30-pre" targetFramework="net45" />
+  <package id="Microsoft.VisualStudio.Composition" version="15.8.98" targetFramework="net45" />
```

Comment:
Why is this needed?

Reply:
I was getting some build errors because the latest version of CPS references this (newer) version of Composition.